.github/workflows/bicep.rb.tpl

```diff
@@ -2,13 +2,29 @@ class Bicep < Formula
   desc "Bicep: next generation template language for Azure Resource Manager (ARM)"
   homepage "https://github.com/Azure/bicep"
   version "${release_version}"
-  url "https://github.com/Azure/bicep/releases/download/v${release_version}/bicep-osx-x64"
-  sha256 "${release_sha256}"
+
+  case
+  when OS.mac? && Hardware::CPU.intel?
```

Comment:
For versions < 0.16.1, we'll want to continue to use the x64 release - could you update this logic to preserve the existing behavior? A simple regex should suffice - e.g. `0\.([1-9]|1[1-5])\.\d+`

Reply:
Hi Anthony, I understand what you mean and I was just modifying the code to add the regex check for versions before 1.16.1. Until I realised:

Homebrew is designed to provide always the latest version of a package. This repository is always fetching the latest version of Bicep before it creates a new Formula, so from now on it is never going to attempt creating a Homebrew Formula for a version older than 1.16.1. Therefore such a regex check would be unnecessary in my opinion.

I triggered again the github action on my forked repository and it is updating the Formula for 1.16.1: https://github.com/cagataygurturk/homebrew-bicep/pull/2

Reply:
Ah, that's a great point!